utils/sample/src/androidTest/java/com/microsoft/device/dualscreen/testing/sample/MockFoldingFeatureSample.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.testing.sample
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.rules.activityScenarioRule
+import com.microsoft.device.dualscreen.testing.DeviceModel.SurfaceDuo
+import com.microsoft.device.dualscreen.testing.DeviceModel.SurfaceDuo2
+import com.microsoft.device.dualscreen.testing.filters.MockFoldingFeature
+import com.microsoft.device.dualscreen.testing.filters.MockFoldingFeature.FoldingFeatureOrientation.HORIZONTAL
+import com.microsoft.device.dualscreen.testing.filters.MockFoldingFeature.FoldingFeatureState.FLAT
+import com.microsoft.device.dualscreen.testing.filters.TargetDevices
+import com.microsoft.device.dualscreen.testing.rules.FoldableTestRule
+import com.microsoft.device.dualscreen.testing.rules.foldableRuleChain
+import com.microsoft.device.dualscreen.testing.runner.FoldableJUnit4ClassRunner
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.RunWith
+
+@RunWith(FoldableJUnit4ClassRunner::class)
+class MockFoldingFeatureSample {
+    private val activityScenarioRule = activityScenarioRule<MainActivity>()
+    private val foldableTestRule = FoldableTestRule()
+
+    @get:Rule
+    val testRule: TestRule = foldableRuleChain(activityScenarioRule, foldableTestRule)
+
+    @Test
+    @MockFoldingFeature(
+        windowBounds = [0, 0, 1000, 1000],
+        size = 10,
+        state = FLAT,
+        orientation = HORIZONTAL
+    )
+    @TargetDevices(ignoreDevices = [SurfaceDuo, SurfaceDuo2])
+    fun testOn10PxFoldingFeature() {
+        onView(withId(R.id.text_view_start)).check(matches(isDisplayed()))
+        onView(withId(R.id.text_view_start)).check(matches(withText(R.string.dual_landscape_start)))
+
+        onView(withId(R.id.text_view_end)).check(matches(isDisplayed()))
+        onView(withId(R.id.text_view_end)).check(matches(withText(R.string.dual_landscape_end)))
+    }
```

Comment:
Can we please have also a sample test without `windowBounds` when using `@MockingFoldingFeature` annotation? I think it will be helpful to higlight that this is not needed and that by default the whole display area is used.

Reply:
Done